[2025-01-06T10:34:18.638+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-06T10:34:18.672+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: kaggle_dag.creating_schema_stg manual__2025-01-06T10:34:14.497188+00:00 [queued]>
[2025-01-06T10:34:18.693+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: kaggle_dag.creating_schema_stg manual__2025-01-06T10:34:14.497188+00:00 [queued]>
[2025-01-06T10:34:18.695+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-01-06T10:34:18.725+0000] {taskinstance.py:2889} INFO - Executing <Task(PostgresOperator): creating_schema_stg> on 2025-01-06 10:34:14.497188+00:00
[2025-01-06T10:34:18.736+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=435) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-06T10:34:18.739+0000] {standard_task_runner.py:72} INFO - Started process 436 to run task
[2025-01-06T10:34:18.739+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'kaggle_dag', 'creating_schema_stg', 'manual__2025-01-06T10:34:14.497188+00:00', '--job-id', '801', '--raw', '--subdir', 'DAGS_FOLDER/kaggle_dag.py', '--cfg-path', '/tmp/tmpiki2hbhx']
[2025-01-06T10:34:18.742+0000] {standard_task_runner.py:105} INFO - Job 801: Subtask creating_schema_stg
[2025-01-06T10:34:18.822+0000] {task_command.py:467} INFO - Running <TaskInstance: kaggle_dag.creating_schema_stg manual__2025-01-06T10:34:14.497188+00:00 [running]> on host 4f9b54363226
[2025-01-06T10:34:19.004+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='kaggle_dag' AIRFLOW_CTX_TASK_ID='creating_schema_stg' AIRFLOW_CTX_EXECUTION_DATE='2025-01-06T10:34:14.497188+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-06T10:34:14.497188+00:00'
[2025-01-06T10:34:19.006+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-06T10:34:19.027+0000] {sql.py:278} INFO - Executing: CREATE SCHEMA IF NOT EXISTS stg;

CREATE TABLE IF NOT EXISTS stg.adv (
daily_time_spent_on_site float,
age SMALLINT,
area_income float,
daily_internet_usage float,
ad_topic_line VARCHAR,
city VARCHAR,
male SMALLINT,
country VARCHAR,
timestamp timestamp,
clicked_on_ad SMALLINT
);
[2025-01-06T10:34:19.044+0000] {base.py:84} INFO - Retrieving connection 'pg_connect'
[2025-01-06T10:34:19.058+0000] {base.py:84} INFO - Retrieving connection 'pg_connect'
[2025-01-06T10:34:19.066+0000] {sql.py:553} INFO - Running statement: CREATE SCHEMA IF NOT EXISTS stg;

CREATE TABLE IF NOT EXISTS stg.adv (
daily_time_spent_on_site float,
age SMALLINT,
area_income float,
daily_internet_usage float,
ad_topic_line VARCHAR,
city VARCHAR,
male SMALLINT,
country VARCHAR,
timestamp timestamp,
clicked_on_ad SMALLINT
);, parameters: None
[2025-01-06T10:34:19.086+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-06T10:34:19.087+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=kaggle_dag, task_id=creating_schema_stg, run_id=manual__2025-01-06T10:34:14.497188+00:00, execution_date=20250106T103414, start_date=20250106T103418, end_date=20250106T103419
[2025-01-06T10:34:19.118+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-06T10:34:19.171+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-01-06T10:34:19.176+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
